src/Styleguide/Components/EntityOverline.tsx

```diff
@@ -0,0 +1,78 @@
+import { Avatar, Box, Flex, Sans, Serif } from "@artsy/palette"
+import FollowArtistButton from "Components/FollowButton/FollowArtistButton"
+import React, { Component } from "react"
+// import { AvatarInitials } from "Styleguide/Components/AvatarInitials"
+
+interface EntityOverlineProps {
+  imageUrl?: string
+  initials?: string
+  meta: string
+  name: string
+  showFollow: boolean
+}
+
+export class EntityOverline extends Component<EntityOverlineProps> {
+  static defaultProps: Partial<EntityOverlineProps> = {
+    showFollow: true,
+  }
+
+  render() {
+    const { imageUrl, initials, name, meta, showFollow } = this.props
+
+    return (
+      <>
+        <Flex alignItems="center">
+          <Box mr={1} pt={0.5}>
+            <Avatar size="xs" src={imageUrl} initials={initials} />
+          </Box>
+
+          <Flex flexDirection="column" justifyContent="center">
+            <Serif size="3" weight="semibold" color="black100">
+              {name}
+            </Serif>
+
+            <Flex>
+              <Sans size="2" color="black60">
+                {meta}
+              </Sans>
+
+              {showFollow && (
```

Comment:
So if meta can be null - in this case we have to test for both to determine if we need to insert `•` between